samples/cpp/benchmark_app/main.cpp

```diff
@@ -414,6 +414,17 @@ int main(int argc, char* argv[]) {
                 set_infer_precision();
             } else if (device.find("AUTO") != std::string::npos) {
                 device_nstreams.erase(device);
+            } else if (device.find("MULTI") != std::string::npos) {
+                setThroughputStreams();
+                set_infer_precision();
+                if ((device_name.find("GPU") != std::string::npos) && (device_name.find("CPU") != std::string::npos)) {
```

Comment:
Remove the logic in GPU branch, move the logic for MULTI in CPU branch here as well.